### PR TITLE
Feature/central deploy repository

### PR DESCRIPTION
**Pull request by ivanmrva**
first version - migrated IST state - DEV deployment running
not quite sure how to organize it in folders (i would need something like ansible actually for the future) - so changes to be follow..

### CODE REVIEW

File: dev/apps.yml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        networks:
+            - jobroom_data2
+        secrets:
+            - source: bootstrap.yml-1
+              target: bootstrap.yml
+        deploy:
+            placement:
+                constraints: [node.labels.type == java]
+#
+#    jobroom-app-eiam:
```
Comment (birom) [inline dev/apps.yml]: We do not need this. This part should have been removed from the original file as well.

File: dev/apps.yml [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+        networks:
+            - jobroom_data2
+        secrets:
+            - source: bootstrap.yml-1
+              target: bootstrap.yml
+        deploy:
+            placement:
+                constraints: [node.labels.type == java]
+
+#    jobpublicationservice-app:
```
Comment (birom) [inline dev/apps.yml]: Please also remove this. The jobpublication service is not used any more.

File: dev/apps.yml [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+#        networks:
+#            - jobroom_data2
+#        secrets:
+#            - source: bootstrap.yml-1
+#              target: bootstrap.yml
+#        deploy:
+#            placement:
+#                constraints: [node.labels.type == java]
+
+    jobservice-app:
```
Comment (birom) [inline dev/apps.yml]: Jobservice is no more used. We can remove it.

File: dev/data-services.yml [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+        networks:
+            - jobroom_data2
+        secrets:
+            - postgres-passwd
+        command: postgres -c tcp_keepalives_idle=60 -c tcp_keepalives_interval=60 -c tcp_keepalives_count=60
+        deploy:
+            placement:
+                constraints: [node.labels.type == data]
+
+    jobservice-postgres:
```
Comment (birom) [inline dev/data-services.yml]: Not needed

File: dev/monitor.yml [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+            ELASTICSEARCH_URL: http://jhipster-elasticsearch:9200
+        ports:
+            - 5602:5601
+        depends_on:
+            - jhipster-elasticsearch
+        deploy:
+            placement:
+                constraints: [node.labels.elk == apps]
+
+    #jhipster-import-dashboards:
```
Comment (birom) [inline dev/monitor.yml]: Please remove the not used sections.

Comment (ivanmrva) [reply dev/monitor.yml]: done

File: dev/deploy-docker-stack.sh [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+docker secret create dev_job-room_ch.crt ./secrets/dev.job-room.ch.pem || true
+docker secret create dev_job-room_ch.key ./secrets/dev.job-room.ch.key || true
+POSTGRES_PWD=$(jq -r '."postgres-passwd"' ./secrets/secret-vars.json)
+echo $POSTGRES_PWD | docker secret create postgres-passwd - || true
+
+echo "Deploying docker stack 'monitor'..."
+docker stack deploy --prune --with-registry-auth --compose-file ./monitor.yml monitor
+
+echo "Deploying docker stack 'infra'..."
+docker stack deploy --prune --with-registry-auth --compose-file ./infr-services.yml infra
```
Comment (birom) [inline dev/deploy-docker-stack.sh]: Must be ```infra.yml```.

File: dev/deploy-docker-stack.sh [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+echo "Deploying docker stack 'app-data'..."
+docker stack deploy --prune --with-registry-auth --compose-file ./data-services.yml app-data
```
Comment (birom) [inline dev/deploy-docker-stack.sh]: Must be ```app-data.yml```

File: dev/deploy-docker-stack.sh (hunk 6 above)
Comment (ivanmrva) [reply dev/deploy-docker-stack.sh]: yep :)

File: dev/deploy-docker-stack.sh (hunk 7 above)
Comment (ivanmrva) [reply dev/deploy-docker-stack.sh]: yep

**Pull request merged**